plugins/inputs/traefik/README.md

```diff
@@ -0,0 +1,87 @@
+# Telegraf Input Plugin: Traefik
+
+This plugin gather health check status from services registered in Traefik. Traefik must be started with the `--api` option (formerly `--web`). See the [configuration docs](https://docs.traefik.io/configuration/api/) for more information.
+
+### Configuration:
+
+Simple configuration, with one measurement per request capture:
+
+```toml
+# Description
+[[inputs.traefik]]
+  ## Required Traefik server address, host and port (default: "127.0.0.1")
+  # address = "http://127.0.0.1:8080"
+  #
+  # [inputs.traefik.tags]
+  #  instance = "prod"
+```
+
+Configuration with additional measurements for each status code encountered:
+
+```toml
+# Description
+[[inputs.traefik]]
+  ## Required Traefik server address, host and port (default: "127.0.0.1")
+  # address = "http://127.0.0.1:8080"
+  # include_status_code_measurement = true
+  #
+  # [inputs.traefik.tags]
+  #  instance = "prod"
+```
+
+### Measurements & Fields:
+
+- `traefik` measurement have the following fields:
+    - **total_count** (int) - total number of responses since the application was last restarted
+    - **average_response_time_sec** (float64) - average time (in seconds) of all responses
+    - **total_response_time_sec** (float64) - total time (in seconds) of all responses
+    - **unixtime** (int) - unix timestamp of when this service was last restarted
```

Comment:
Is this description accurate?  Traefik's docs say ` // current server date in seconds`.

At any rate, for times we are standardizing on saving them as integers in nanoseconds since the epoch.  I don't think we need to call it unixtime, something descriptive of the correct definition should work: `server_time` or ``startup_time`